swift/Tests/SwiftNoosphereTests/NoosphereTests.swift

```diff
@@ -553,10 +553,10 @@ final class NoosphereTests: XCTestCase {
             print("Authorization:", authorization)
             
             ns_sphere_save(noosphere, sphere, nil, nil)
-            
-            nsSphereAuthorityEscalate(noosphere, sphere, sphere_mnemonic_ptr) {
-                (error, root_sphere) in
-                
+
+            nsSphereAuthorityAuthorizationsList(noosphere, sphere) {
+                (error, authorizations) in
```

Comment:
I believe this `authorizations` (an array of owned-strings) will need to be freed